assets/templates/partials/json-ld/dataset/legacy.tmpl

```diff
@@ -1,16 +1,17 @@
 {{ $data := .DatasetLandingPage }}
+{{ $URLPath := .URI }}
     {{ $length := len $data.Datasets }}{{ if gt $length 0 }}
         {{ $latestDataset := index $data.Datasets 0}}
         {{ $numberOfDownloads := len $latestDataset.Downloads }}
         "distribution": [
         {{ range $i, $el := $latestDataset.Downloads }} {
             "@type": "DataDownload",
             "encodingFormat": {{ .Extension }},
-            "contentUrl": {{ .URI }}
+            "contentUrl": {{concatenateStrings "www.ons.gov.uk/file?uri=" $URLPath "/current/" .URI }}
```

Comment:
I think it would be better to use the `SiteDomain` here as well rather than being hard coded.

Reply:
Have updated to use `SiteDomain` and also a `SubDomain` value derived from the site language